semver/format.ts

```diff
@@ -12,6 +12,10 @@ function formatNumber(value: number) {
   }
 }
 
+/**
+ * @deprecated (will be removed in 0.213.0)
+ */
+export function format(semver: SemVer, style?: FormatStyle): string;
```

Comment:
Having an overload signature that's identical to the implementation signature doesn't make sense. Did you mean to create two overload signatures and deprecate the one that contains the `style` argument?